<launch>

    <!-- Start the Dynamixel Controller -->
    <node name="dynamixel_manager" pkg="dynamixel_controllers" type="controller_manager.py" required="true" output="screen">
        <rosparam file="$(find third_arm)/config/manager.yaml" command="load" />
    </node>

    <rosparam file="$(find third_arm)/config/dof_controllers.yaml" command="load"/>

    <!-- Start Joint Position Controllers for each DoF -->
    <node name="ctrl_spawner" pkg="dynamixel_controllers" type="controller_spawner.py"
          args="--manager=dxl_manager
                --port=third_arm_port
                 base_swivel_controller
                 vertical_tilt_controller
                 arm_extension_controller
                 wrist_controller
		 gripper_controller"
          output="screen">

    </node>

</launch>
